.github/workflows/stale.yaml

```diff
@@ -0,0 +1,30 @@
+name: Label stale PRs and issues
+
+on:
+  schedule:
+    - cron: "30 1 * * *" # Runs daily at 1:30 AM UTC
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/stale@v9
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-issue-stale: 30
+          days-before-issue-close: -1 # never automatically close stale issues
```

Comment:
@davidfestal should we mark PRs stale after 14 days of inactivity, and close them after 30 days? Will that mean they get recreated by your crontabs?

Reply:
> @davidfestal should we mark _issues_ like #982 stale after 14 days of inactivity, and close them after 30 days? 

I think that the proposed rule of simply marking them stale after 30 days and not closing them is good for now.
Please see my general comment about why: https://github.com/redhat-developer/rhdh-plugin-export-overlays/pull/1030#pullrequestreview-2806532411

> Also why are we using issues here, when we have JIRA? 😕

I created it just to track that this PR was failing, stick the result of my first analysis, and take the opportunity to add a `help wanted` label.
This doesn't prevent us from adding JIRA issues related to the overlay repository.